README.md

```diff
@@ -22,8 +22,8 @@ Using service manager (sm or sm2)
 Use the ARS_ALL profile to bring up all services using the latest tagged releases
 ``sm2 --start ARS_ALL``
 
-run `sm2 -s` to check what services are running
-###Launching the service locally
+Run `sm2 -s` to check what services are running.
+Launching the service locally
```

Comment:
@ymabdulahi I think there should be a blank line before `###Launching the service locally` for the formatting to take effect.

i.e. like below:
```
Run `sm2 -s` to check what services are running.

###Launching the service locally
```